docs/api/webxrHardwareExtensionAPIs.md

```diff
@@ -1,16 +1,42 @@
----
+.---
 title: WebXR Hardware Extension APIs
 type: api
 layout: docs
 order: 2
 parent_section: api
 ---
 
-This currently consists of seperate device-specific APIs, but will be folded into a more generic API, properly coded in Exokit generically on the `session` objects rather than under a `magicleap` namespace.
+The WebXR Hardware Extension APIs is a generic API that is not tied to any one hardware platform and provides access to: meshing, planes detection, hand tracking, eye tracking. This is inspired by the [WebExtension API style](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API). Although it is not a web standard yet, these are finding their way into the WebXR spec and Exokit wants to be standards-compliant.
+
+No hardware is needed to test these APIs as Exokit Studio provides emulation and emulated data.
+
+# Basic usage
+
+Request a `session` instance with the `display.requestSession()` method in order to interact with XR device's presentation or tracking capabilities. Specify under `extensions` which APIs you want to access:
+
+```js
+const session = await display.requestSession({
+  exclusive: true,
+  extensions: {
+    meshing: true,
+    planesTracking: true,
+    handTracking: true,
+    eyeTracking: true,
+  },
+});
+  session.addEventListener('meshadd', _meshadd);
+  session.addEventListener('meshupdate', _meshupdate);
+  session.addEventListener('meshremove', _meshremove);
+```
+
+When the API mechanism for an `InputSource` is invoked, events are fired on `session`.
+
+Calling `getInputSources()` on `display.session` will return a list of all `InputSources` that are currently active.
+
 
 ## Magic Leap One API
 
-The Magic Leap API is exposed to sites under the `window.browser.magicleap` endpoint. This is inspired by the [WebExtension API style](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API). It is not a web standard (yet).
+The Magic Leap API is exposed to sites under the `window.browser.magicleap` endpoint.
```

Comment:
This won't be true once the above is true.

Reply:
Yes I'll update now that #1052  has the magic leap hardware bindings to the new WebXR extension APIs.

Reply:
Removed in 7bf70fa